astropy/convolution/tests/test_kernel_class.py

```diff
@@ -274,26 +274,26 @@ def test_rmultiply_scalar_type(self, number):
     def test_multiply_kernel1d(self):
         """Test that multiplying two 1D kernels raises an exception."""
         gauss = Gaussian1DKernel(3)
-        with pytest.raises(Exception):
+        with pytest.raises(Exception, match="Kernel operation not supported."):
```

Comment:
Updates are needed (after rebasing) to use https://github.com/astropy/astropy/pull/14728, e.g.:
```suggestion
        with pytest.raises(KernelArithmeticError, match="Kernel operation not supported."):
```

Reply:
Might be easier if a human do this update from their own fork by cherry-picking commit from here and then rebasing?

Reply:
Or a quick followup PR. adding the `match="Kernel operation not supported` does not affect the Exception type.

Reply:
@larrybradley, is a quick followup alright?

Reply:
yes, thanks.